res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -1085,14 +1052,28 @@ NumarkMixtrack3.init = function(id, debug) {
     print("");
 
     // Set soft-takeover for all Sampler volumes
-    for (i = engine.getValue("[Master]", "num_samplers"); i >= 1; i--) {
+    for (var i = engine.getValue("[Master]", "num_samplers"); i >= 1; i--) {
         engine.softTakeover("[Sampler" + i + "]", "pregain", true);
     }
 
-    for (i = engine.getValue("[Master]", "num_decks"); i >= 1; i--) {
+    for (var i = engine.getValue("[Master]", "num_decks"); i >= 1; i--) {
```

Comment:
I'm aware that JS lacks block scoping when using `var`, and that variable declarations get hoisted, but I think pretty much everyone except Crockford will advocate for including var in for loops. 

As an aside, the mapping in its current state is not functional since I committed stuff without having the controller on hand. I will go back and test things commit by commit though!